src/token/stream-parser.js

```diff
@@ -19,6 +20,9 @@ tokenParsers[TYPE.ROW] = require('./row-token-parser');
 tokenParsers[TYPE.NBCROW] = require('./nbcrow-token-parser');
 tokenParsers[TYPE.SSPI] = require('./sspi-token-parser');
 
+const pow2_32 = Math.pow(2, 32);
+const maxInt64HiVal = Math.pow(2, Math.floor(Math.log2(Number.MAX_SAFE_INTEGER)) - 32);
```

Comment:
Hi @SergeS, Thanks for making this change. Just wondering, for this line, are you trying to extract the high bits for the MAX_SAFE_INTEGER, so you can use it later to compare with the inputs's high bits value? Is this the case, or I misunderstood your purpose for this?

Reply:
Yes, exactly, it is used to check high 32 bits of int64 later and depending on result either parse as number or as string